wails-frontend/src/pages/backupMethods.tsx

```diff
@@ -0,0 +1,46 @@
+import { promptRequest } from '../events/events';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { Button } from '@massalabs/react-ui-kit';
+
+function BackupMethods() {
+  const navigate = useNavigate();
+
+  const { state } = useLocation();
+  const req: promptRequest = state.req;
+
+  const baselineStr = 'Choose a back-up method';
+
+  function downloadYml() {
+    console.log('Download a .yaml file');
```

Comment:
Is that here because BE will handle after!? 🤔

Reply:
yes